README.md

```diff
@@ -1,30 +1,26 @@
 # Zed [![Tests][tests-img]][tests]
 
-The Zed project is a new, clean-slate design for a data engineering stack.
-At Zed's foundation lies a new family of self-describing
-data formats based on the "Zed data model", which blends the highly structured
-approach of dataframes and relational tables with the loosely structured
-document model of JSON.
+Zed is an open-source, cloud-native, and searchable data lake for
```

Comment:
```suggestion
Zed is an open-source, cloud-native, and searchable data lake platform for
```

Does the addition of the word "platform" makes this too word salad? Reading this while pretending to be a user seeing it for the first time, it read a little funny to me without it because once you drop the adjectives it shortens to "Zed is a data lake". While "Zed lakes" are a thing, they're enabled by the platform (the Zed code, which lives in a Zed repo, which is delivered through the Zed tools).